src/Mod/Path/PathScripts/PathAdaptiveGui.py

```diff
@@ -78,13 +78,22 @@ def getForm(self):
 
         # helix angle
         form.HelixAngle = QtGui.QDoubleSpinBox()
-        form.HelixAngle.setMinimum(0.1)
-        form.HelixAngle.setMaximum(90)
-        form.HelixAngle.setSingleStep(0.1)
-        form.HelixAngle.setValue(5)
+        form.HelixAngle.setMinimum(0.5)
+        form.HelixAngle.setMaximum(359)
+        form.HelixAngle.setSingleStep(1)
+        form.HelixAngle.setValue(180)
```

Comment:
These values are a bit confusing. I would expect the helix angle to be a value between 0.1 and 90 where 0.1 is a very shallow helix and 90 is a normal z move?

Reply:
![HelixAngle](https://user-images.githubusercontent.com/1136710/104458469-59d39200-55a3-11eb-98d4-24c305674a42.png)

Reply:
https://en.wikipedia.org/wiki/Helix_angle

Reply:
I changed it to 360° for a full circle, but i can change it back to 90